### DISCUSSION TITLE
Ideas on combining table refresh with NTIE and sorting logic

### DISCUSSION THREAD

**Discussion by podliashanyk** [Uninett/Howitz#96] [Ideas]
Table refresh with NTIE in #87 does not reorder events. Meaning there is no consideration towards the sorting logic (see #81). New events are simply prepended, modified events always stay where they were in the table before NTIE.

**Comment by hmpf** [comment]
Yeah the python-code should either insert changed events into the correct order or we need to redraw the entire list regularly. Curitz does the latter but has to do that due to how curses works I suspect. The code for this is in curitz itself, zinolib knows nothing about sort order.

**Reply by podliashanyk** [reply]
NB that inserting events into correct order would also require some advanced fickling with `hx-swap` and `hx-target` attributes. We might even need to define a custom event for that (JS). In short: single `hx-swap` and `hx-target` on `tbody` will **definitely** be replaced by individual ones on each row, and each row would **probably** listen to the custom event. 

Seems easier to have python-code for figuring out the sorting order and then reswapping **all** of the table rows if there are any changes to sorting. As a plus this option might also work well in tandem with https://github.com/Uninett/Howitz/discussions/97 if we land on some solution with reswapping all of the rows there also.

**Comment by podliashanyk** [comment]
Fixed by using cache in #81
